apps/gidd/schema.py

```diff
@@ -137,11 +155,29 @@ class Query(graphene.ObjectType):
         )
     )
     gidd_release_meta_data = graphene.Field(GiddReleaseMetadataType)
+    gidd_public_country_list = graphene.NonNull(graphene.List(GiddPublicCountryType))
```

Comment:
Shouldn't this be like this?

```suggestion
    gidd_public_country_list = graphene.List(graphene.NonNull(GiddPublicCountryType))
```